azext_iot/_help.py

```diff
@@ -36,6 +36,10 @@
     long-summary: |
                   This command relies on and may install dependent Cython package (uamqp) upon first execution.
                   https://github.com/Azure/azure-uamqp-python
+
+                  Note: The event will be displayed even if the message body is non-unicode decodable, in this case
+                  the event payload portion will be displayed as {{non-unicode decodable content}} with rest of the
```

Comment:
@digimaun I'm actually using {{non-unicode decodable payload}} in monitor event and {{non-decodable payload}} in c2d-message receive since I thought the scenario was a little different:

in d2c we show the true payload content as long as it's unicode presentable disregarding the ce user entered
in c2d we only show the true payload content when the ce falls under unicode category and it's decodable when using the ce provided

do you think it's better to keep consistency between c2d and d2c? in that case I will update both to {{non-decodable content}}, thanks!